src/sdk/namespace_fs.js

```diff
@@ -2450,7 +2445,6 @@ class NamespaceFS {
      * @returns {string}
      */
      _get_version_entry_key(dir_key, ent) {
-        if (ent.name === config.NSFS_FOLDER_OBJECT_NAME) return dir_key;
```

Comment:
where is the fix of the object name when running list_versions and it's a directory object?

Reply:
there is no special case for versioned object. we get the .folder file. in the end we format the key to remove the ,folder from the name. also its already in the correct directory level in case of delimiter so no need to change anything